internal/refreshtoken/refresh_token.go

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+// The refreshtoken package encapsulates domain logic surrounding
```

Comment:
```suggestion
// Package refreshtoken encapsulates domain logic surrounding
```
The Go convention for package comments is for the first sentence to start with "Package <package name>"